opm/parser/eclipse/EclipseState/Tables/PvtgTable.hpp

```diff
@@ -1,5 +1,5 @@
 /*
-  Copyright (C) 2013 by Andreas Lauser
+  Copyright (C) 2014 by Andreas Lauser
```

Comment:
these notes are more or less useless IMO: they are not legally binding (the copyright is at the person/company who wrote the code, regardless of what the header says) and they are a pain in the a** to keep up to date (thus, they are often wrong). Possibly we should just remove them anywhere?